### PR TITLE
Fix #7: add sitemap generation

### LINKED ISSUE #7: Generate sitemaps.xml

### PR DESCRIPTION
**Pull request by Thooms**

### CODE REVIEW

**Comment by Thooms** [comment]
ping @florent1933 @MrChoclate review

**Comment by florent1933** [comment]
@MrChoclate : Please review the code

File: generate.py [hunk 1]
```diff
@@ -79,6 +82,30 @@ def render_item(self, slug, entry):
...
+            }
+            for page in ('conferences', 'trainings', 'talks')
+        ])
+        pages.extend([
+            {
+                'url': self.domain + '/{}.html'.format(slug),
+                'lastmod': datestr
+            }
+            for slug in self.items
+        ])
```
Comment (MrChoclate) [inline generate.py]: I smell duplication.

for page in ( ... ) + self.items?

Why use pages = []
I would first start with comprehension list, then add the domain url.

Comment (flavray) [inline generate.py]: I think it is more « clear » like this, you know exactly what happens. Using `for page in ( ... ) + self.items` is a bit more cryptic and dirty

Comment (MrChoclate) [inline generate.py]: Well if you want to specify another field like priority or changefreq and they are different ok. Otherwise it is duplication. DRY.

**Pull request merged**